applications_tests/gls_navier_stokes_2d/gls_vof_wetting_mesh-adapt.prm

```diff
@@ -0,0 +1,186 @@
+subsection multiphysics
+	set VOF = true
+end
+
+#---------------------------------------------------
+# VOF
+#---------------------------------------------------
+subsection VOF	
+	subsection peeling wetting
+		set enable = true
+		set verbosity = verbose
+		set diffusivity = 5e-3
+	end
+	subsection interface sharpening
+		set enable = true
+		set frequency   = 1
+		set interface sharpness    = 2
+		set threshold   = 0.5
+	end
+end
+
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+subsection initial conditions
+    set type = nodal
+    subsection uvwp
+            set Function expression = 0; 0; 0
+    end
+    subsection VOF
+	set Function expression = if ( (x>-0.06)&&(x<0.06)&&(y<0.1)&&(y>0.01) ,1,0)
+    end
+end
+
+#---------------------------------------------------
+# Source term
+#---------------------------------------------------
+subsection source term
+    set enable = true
+
+    subsection xyz
+        set Function expression = 0; -10; 0
+    end
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+  set number of fluids     = 2
+  subsection fluid 1
+    set density              = 1
+    set kinematic viscosity  = 0.01
+  end
+  subsection fluid 0
+    set density              = 0.005
+    set kinematic viscosity  = 0.1
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+		set type = dealii
+		set grid type = hyper_rectangle
+		set grid arguments = -0.1, 0 : 0.1, 0.2 : true
+		set initial refinement = 5
+end
+
+# --------------------------------------------------
+# Analytical Solution
+#---------------------------------------------------
+subsection analytical solution
+  set enable     	= false
+  set verbosity 	= quiet
+end
+
+#---------------------------------------------------
+# Timer
+#---------------------------------------------------
+subsection timer
+   set type = none
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 4
+    subsection bc 0
+        set id = 0
+        set type              = slip
+    end
+    subsection bc 1
+	set id = 1
+        set type              = slip
+    end
+    subsection bc 2
+        set id = 2
+        set type              = noslip
+    end
+    subsection bc 3
+        set id = 3
+        set type              = noslip
+    end
+end
+
+subsection boundary conditions VOF
+  set number                  = 4
+    subsection bc 2 #lower wall
+	set id = 2
+        set type              = peeling/wetting
+    end
+    subsection bc 3 #upper wall
+	set id = 3
+        set type              = peeling/wetting
+    end
```

Comment:
This is weird. If you only have two active bcs you should set number  = 2 and not 4... this does not follow the regular syntax. We only specify the BCs we need.

Reply:
Yes you are right, this was from previous testing on other BC as well, I will change that